skaffold-duct-tape/duct-tape/Dockerfile

```diff
@@ -19,3 +24,4 @@ WORKDIR /duct-tape
 COPY --from=delve /go/bin/dlv go/bin/
 COPY --from=python27 /ptvsd/ python/
 COPY --from=python37 /ptvsd/ python/
+COPY --from=netcore /vsdbg/ netcore/
```

Comment:
Will vsdbg support future versions like .NET 5?  Should we version this directory?

Reply:
`vsdbg` seems to work with all versions of .NET Core so far (1.0, 2.0 and soon to be released 3.0) so I don't think that will be a problem.

Reply:
That being said, I just realized `vsdbg` is runtime specific so this version of `vsdbg` will only work for Linux container.

Reply:
That's true of the python support files too.  We'll have to publish separate images when/if we support alternative os/arch combinations.